lib/util/github/tags.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { GithubHttp } from '../http/github';
+import * as githubGraphql from './graphql';
+import { findCommitOfTag } from './tags';
+
+describe('util/github/tags', () => {
+  describe('findCommitOfTag', () => {
+    let http: GithubHttp;
+
+    beforeEach(() => {
+      http = new GithubHttp();
```

Comment:
does it change something when we don't recreate it for each test? let's make it static if possible.

Reply:
the only difference is that it will run even if no specs from this file are selected. Will move to a constant